content/getting-started/install/macos.md

```diff
@@ -9,7 +9,7 @@ description: >
 
 This page has information on how to install and use TinyGo on macOS. If you wish to build TinyGo from source, for example if you intend to contribute to the project, please take a look [here](../../../docs/guides/build).
 
-You must have Go v1.15+ already installed on your machine in order to install TinyGo. We recommend Go v1.17+.
+You must have Go v1.16+ already installed on your machine in order to install TinyGo. We recommend Go v1.17+.
```

Comment:
We probably should also "recommend" `v1.18+`

Reply:
I can change it but the other versions are recommending 1.17+.

Reply:
Maybe we should change them all? Probably yes.